e2e/04-payment-send-flow.e2e.spec.ts

```diff
@@ -1,12 +1,116 @@
+import { bech32 } from "bech32"
 import { i18nObject } from "../app/i18n/i18n-util"
 import { loadLocale } from "../app/i18n/i18n-util.sync"
-import { selector } from "./utils"
+import { selector, goBack } from "./utils"
 import { getInvoice } from "./utils/graphql"
 
-describe("Payments Flow", () => {
-  loadLocale("en")
-  const LL = i18nObject("en")
-  const timeout = 30000
+loadLocale("en")
+const LL = i18nObject("en")
+const timeout = 30000
+
+describe("Lnurl Pay Flow", () => {
+  // see https://github.com/Extheoisah/lnurl-json for reference to lnurl json
+  const words = bech32.toWords(
```

Comment:
do we have a test for lightning address? if you can try to add `extheo@testlnurl.netlify.app` also. can be done in another PR

Reply:
Yeah. Will do this in another pr